traitsui/testing/tester/wx/helpers.py

```diff
@@ -55,6 +55,83 @@ def mouse_click_checkbox(control, delay):
     control.ProcessWindowEvent(click_event)
 
 
+def mouse_click_combobox_or_choice(control, index, delay):
+    """ Performs a mouce click on either a wx combo box or a wx choice on the
+    entry at the given index.
+
+    Parameters
+    ----------
+    control : wxComboBox or wxChoice
+        The wx Object to be clicked.
+    index : int
+        The index of the item in the combobox/choice to be clicked
+    delay: int
+        Time delay (in ms) in which click will be performed.
+    """
+    if not control.IsEnabled():
+        return
+    wx.MilliSleep(delay)
+    if isinstance(control, wx.ComboBox):
+        click_event = wx.CommandEvent(
+            wx.wxEVT_COMMAND_COMBOBOX_SELECTED, control.GetId()
+        )
+    elif isinstance(control, wx.Choice):
+        click_event = wx.CommandEvent(
+            wx.wxEVT_COMMAND_CHOICE_SELECTED, control.GetId()
+        )
+    else:
+        raise TypeError
```

Comment:
I did not comment on this earlier thinking the refactoring suggested might make it irrelevant: if this stays, please instantiate the exception with an error message.